content/docs/examples/bookinfo/index.md

```diff
@@ -66,42 +66,38 @@ To start the application, follow the instructions corresponding to your Istio ru
 If you use GKE, please ensure your cluster has at least 4 standard GKE nodes. If you use Minikube, please ensure you have at least 4GB RAM.
 {{< /tip >}}
 
-1. Change directory to the root of the Istio installation directory.
-
-1.  Bring up the application containers:
-
-    *   If you are using [manual sidecar injection](/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection),
-        use the following command
-
-        {{< text bash >}}
-        $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo.yaml@)
-        {{< /text >}}
-
-        The `istioctl kube-inject` command is used to manually modify the `bookinfo.yaml`
-        file before creating the deployments as documented [here](/docs/reference/commands/istioctl/#istioctl-kube-inject).
+1.  Change directory to the root of the Istio installation directory.
 
-    *   If you are using a cluster with
-        [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection)
-        enabled, label the `default` namespace with `istio-injection=enabled`
+1.  The default Istio installation uses [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
+    Label the namespace that will host the application with `istio-injection=enabled`:
```

Comment:
How are the previous statement and this one connected? Do readers need to know the default behavior at this time to complete the auto injection? If the sidecar is injected automatically by default, why do I need to add the label to the `default` namespace?

Reply:
It uses automatic injection but it's on a per-namespace, so the namespace still has to be labeled. Does it make sense?